models/census.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var mongoose = require('mongoose');
+
+var censusSchema = mongoose.Schema({
+    'type': String,
+    'activity_year': String,
+    'data': [{
```

Comment:
In the case that we have some proper multi-key indexes defined, there may not be enough of a performance benefit in using the `data` subdoc, especially when it comes to having to use aggregate queries to get at the proper dataset in mongoose. My vote would be then to flatten a bit and create the proper indexes.